README.md

```diff
@@ -42,6 +42,25 @@ Crystalball.foresee do |predictor|
 end
 ```
 
+## Map Generator
+
+There are different map generator strategies that can (and should) be used together for better predictions. Each one has its own benefits and drawbacks, so they should be configured to best fit your needs.
+
+- **CoverageStrategy** uses coverage information to detect which files are covered by the given spec (i.e. the files that, if changed, may potentially break the spec);
+- **AllocatedObjectsStrategy** looks for the files in which the objects created during the spec execution are defined. It is considerably slower than `CoverageStrategy`.
```

Comment:
I believe we need to add more information on how to configure & extend strategies. 
Please mention DI we have and what they are used for, also mention convenience `build` method for `AllocatedObjectsStrategy`. Fully configured example will be useful
Also we need information about Rails views strategy.

Reply:
we definitely need to add more info for all strategies

Reply:
@EPecherkin can you be more specific?